MSAL/src/requests/MSALSilentRequest.m

```diff
@@ -66,19 +66,53 @@ - (id)initWithParameters:(MSALRequestParameters *)parameters
 }
 
 - (void)acquireToken:(MSALCompletionBlock)completionBlock
+{
+    NSString *upn = nil;
```

Comment:
this code looks very duplicate of InteractiveRequest. Can we move it to the base request class?